src/Engine/Manager/RenderManager.cpp

```diff
@@ -258,100 +260,99 @@ void RenderManager::RenderWorldEntities(World& world, const glm::mat4& viewMatri
     const glm::mat4 viewProjectionMatrix = projectionMatrix * viewMatrix;
     const std::vector<Mesh*>& meshComponents = meshes.GetAll();
     //Render shadows maps.
-    {
```

Comment:
Wasn't this indentation explicitly fixed in some other commit? Have we reached a conclusion as to whether we should indent or not?

Reply:
We should indent. But it's also unnecessary to create so many scopes that are the same.

Reply:
Agreed. Is it possible to put the profilers in the same scope? Nesting them makes little difference as they all have the same lifetime. This way we would get away with a single level of indentation.

Reply:
Is it okey to fix indents in another pull request? My next pull request will change this code

Reply:
I'd say so. When you're at it, see if you can place all profilers after each other. That would probably be the cleanest. We still need to define the scope so that they die properly, but that's only one level of indentation then,